src/compas_rhino/geometry/surfaces/nurbs.py

```diff
@@ -0,0 +1,490 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from itertools import groupby
+
+from compas.geometry import Point
+from compas.geometry import NurbsSurface
+
+from compas_rhino.conversions import point_to_rhino
+from compas_rhino.conversions import point_to_compas
+from compas_rhino.conversions import vector_to_compas
+from compas_rhino.conversions import plane_to_compas_frame
+from compas_rhino.conversions import box_to_compas
+from compas_rhino.conversions import RhinoCurve
+
+import Rhino.Geometry
+
+
+def rhino_surface_from_parameters(points, weights, u_knots, v_knots, u_mults, v_mults, u_degree, v_degree, is_u_periodic=False, is_v_periodic=False):
```

Comment:
Nitpicking here, just feel a bit strange of this function hanging here outside with a class, could it be part of class or in a collection of helpers in case it can be re-used elsewhere? (Same goes with RhinoNurbsCurve)